src/Microsoft.OpenApi/Microsoft.OpenApi.csproj

```diff
@@ -6,6 +6,7 @@
         <Version>2.0.0-preview4</Version>
         <Description>.NET models with JSON and YAML writers for OpenAPI specification</Description>
         <SignAssembly>true</SignAssembly>
+        <IsTrimmable Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)','net5.0'))">true</IsTrimmable>        
```

Comment:
Sorry, I forgot to mention that you should probably add this to the csproj too since you have net8.0 targets here.